### PR TITLE
Bring in changes from published v1.0.0 on Chef Server

### PR DESCRIPTION
**Pull request by jeffbyrnes**
@eherot, this set of changes comes from:

1. The copy of singularity v1.0.0 on our Chef Server
2. the `evertrue/eherot/create_install_and_configure_recipes` branch
    * The tests are what come in from the branch

Would you mind taking a look & helping me straighten this out?

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -9,8 +9,8 @@
 default[:singularity] = {
   :user                     => 'singularity',
   :group                    => 'singularity',
-  :git_ref                  => 'Singularity-0.4.1',
-  :version                  => '0.4.1',
+  :git_ref                  => 'e2405eb5ca1a1ba006a89a27bdb3299433ae96d5',
```
Comment (eherot) [inline attributes/default.rb]: Still no.

File: recipes/configure.rb [hunk 2]
```diff
@@ -11,16 +11,15 @@
 
 execute 'migrate_singularity_db' do
   command "#{node[:java][:java_home]}/bin/java " \
-          "-jar #{node[:singularity][:home]}/bin/" \
-          "SingularityService-#{node[:singularity][:version]}-shaded.jar " \
+          "-jar #{node[:singularity][:home]}/bin/singularity.jar " \
```
Comment (eherot) [inline recipes/configure.rb]: This doesn't look right.

File: recipes/configure.rb [hunk 3]
```diff
@@ -11,16 +11,15 @@
...
           "db migrate #{node[:singularity][:conf_dir]}/singularity.yaml " \
           "--migrations #{node[:singularity][:home]}/mysql/migrations.sql " \
           "&& touch #{node[:singularity][:conf_dir]}/migration_ran"
   action  :nothing
 end
 
-cookbook_file "#{node[:singularity][:home]}/mysql/migrations.sql" do
-  source   'migrations.sql'
+remote_file "#{node[:singularity][:home]}/mysql/migrations.sql" do
```
Comment (eherot) [inline recipes/configure.rb]: This is definitely backwards.

**Comment by eherot** [comment]
:-1: :hankey:

**Comment by jeffbyrnes** [comment]
Closing after Slack convo in #ops starting with [this message](https://evertrue.slack.com/archives/ops-dev/p1431625080000665).

**Pull request closed without merging**